pages/price-feeds/use-pyth-for-morpho.mdx

```diff
@@ -8,24 +8,26 @@ Pyth provides a wrapper which implements Morpho's `IOracle` interface called [`p
 
 There are two steps to use Pyth price feeds for Morpho markets:
 
-1. Run the price pusher or scheduler.
+1. Schedule Price Updates.
 2. Deploy the [`MorphoPythOracle.sol`](https://github.com/pyth-network/pyth-morpho-wrapper/blob/main/src/morpho-pyth/MorphoPythOracle.sol) contract for the respective price feed pair.
 
 <Steps>
 
-### Run the price pusher or scheduler
+### Schedule Price Updates
 
 As a pull oracle, Pyth's users are typically responsible for updating the state of on-chain feeds.
 Please see [What is a Pull Oracle?](/price-feeds/pull-updates) to learn more about pull updates.
 
+Consult [Schedule Price Updates](/price-feeds/schedule-price-updates) guide for more information.
+
 The Pyth Data Association sponsors regular on-chain updates for some price feeds.
 See [Sponsored Feeds](./sponsored-feeds.mdx) for the current list of feeds and their update parameters.
 
-If you don't find relevant price IDs in the [Sponsored Feeds](./sponsored-feeds.mdx) list, please contact the Pyth team [here](https://tally.so/r/nGz2jj) to run the scheduler/price pusher for the price feed you need.
+If you don't find relevant price IDs in the [Sponsored Feeds](./sponsored-feeds.mdx) list, please contact the Pyth team [here](https://tally.so/r/nGz2jj) to run the Price Pusher for the price feed you need.
```

Comment:
let's call it Push Feeds. pls check if any other place still uses the term sponsored feeds and replace it.